_posts/2023-04-19-12.0.0-release.md

```diff
@@ -0,0 +1,264 @@
+over 3 months of development work and includes [**XXX resolved issues**][1]
+from [**YYY distinct contributors**][2]. See the [Install Page](https://arrow.apache.org/install/)
+to learn how to get the libraries for your platform.
+
+The release notes below are not exhaustive and only expose selected highlights
+of the release. Many other bugfixes and improvements have been made: we refer
+you to the [complete changelog][3].
+
+## Community
+
+Since the 11.0.0 release, Wang Mingming, Mustafa Akur and Ruihang Xia
+have been invited to be committers.
+Will Jones have joined the Project Management Committee (PMC).
+
+Thanks for your contributions and participation in the project!
+
+## Columnar Format Notes
+
+A first "canonical" extension type has been formalized: `arrow.fixed_shape_tensor` to
+represent an Array where each slot contains a tensor, with all tensors having the same
+dimension and shape. This is based on a Fixed-Size List layout as storage array
+(https://arrow.apache.org/docs/dev/format/CanonicalExtensions.html#fixed-shape-tensor-extension).
+
+## Arrow Flight RPC notes
+
+The JDBC driver for Arrow Flight SQL has had some bugfixes, and has been refactored into a core library (which is not distributed as an uberjar with shaded names) and a driver (which is distributed as an uberjar).
+
+The Java server builder API now offers easier access to the underlying gRPC builder.
+
+Go now implements the Flight SQL extensions for Substrait and transaction support.
+
+## Plasma notes
+
+Plasma was deprecated since 10.0.0. Plasma is removed in this
+release. [GH-33243][GH-33243]
+
+## C++ notes
+
+* Run-End Encoded Arrays have been implemented and are accessible ([GH-32104](https://github.com/apache/arrow/issues/32104))
+* The FixedShapeTensor Logical value type has been implemented using ExtensionType ([GH-15483](https://github.com/apache/arrow/issues/15483), [GH-34796](https://github.com/apache/arrow/issues/34796))
+
+### Compute
+
+* New kernel to convert timestamp with timezone to wall time ([GH-33143](https://github.com/apache/arrow/issues/33143))
+* Cast kernels are now built into libarrow by default ([GH-34388](https://github.com/apache/arrow/issues/34388))
+
+### Acero
+
+* Acero has been moved out of libarrow into it's own shared library, allowing for smaller builds of the core libarrow ([GH-15280](https://github.com/apache/arrow/issues/15280))
+* Exec nodes now can have a concept of "ordering" and will reject non-sensible plans ([GH-34136](https://github.com/apache/arrow/issues/34136))
+* New exec nodes: "pivot_longer" ([GH-34266](https://github.com/apache/arrow/issues/34266)), "order_by" ([GH-34248](https://github.com/apache/arrow/issues/34248)) and "fetch" ([GH-34059](https://github.com/apache/arrow/issues/34059))
+* Reorder output fields of "group_by" node so that keys/segment keys come before aggregates ([GH-33616](https://github.com/apache/arrow/issues/33616))
+
```

Comment:
@westonpace would be useful to add a Substrait section with a summary of the changes/improvements in the Substrait consumer support?

Reply:
Yes, that would be a good idea.  I'll do that shortly.

Reply:
Added here: https://github.com/raulcd/arrow-site/pull/4